src/cmd-kola

```diff
@@ -1,37 +1,56 @@
-#!/usr/bin/env bash
-set -euo pipefail
+#!/usr/bin/env python3
 
-dn=$(dirname "$0")
-# shellcheck source=src/cmdlib.sh
-. "${dn}"/cmdlib.sh
+import argparse
+import subprocess
+import json
+import os
+import sys
 
-latest_qcow=$(get_latest_qemu)
-if [ -z "${latest_qcow}" ]; then
-    fatal "No latest build"
-fi
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+
+import cmdlib
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID")
+parser.add_argument("--output-dir", help="Output directory")
+parser.add_argument("subargs", help="Remaining arguments for kola", nargs='*',
+                    default=['run'])
+args = parser.parse_args()
+
+builds = cmdlib.Builds()
+if args.build is None:
+    args.build = builds.get_latest()
+builddir = builds.get_build_dir(args.build)
+with open(os.path.join(builddir, "meta.json")) as f:
+    buildmeta = json.load(f)
+
+qemuimg = buildmeta['images'].get('qemu')
+if qemuimg is None:
+    raise SystemExit(f"No qemu image in build: {args.build}")
+qemupath = os.path.join(builddir, qemuimg['path'])
 
 # XXX: teach to kola to auto-detect based on prefix; see discussions in
 # https://github.com/coreos/coreos-assembler/pull/85
-args=
-bn=$(basename "${latest_qcow}")
-if [[ ${bn} = fedora-coreos-* ]]; then
-    args="-b fcos"
-elif [[ ${bn} = rhcos-* ]]; then
-    args="-b rhcos"
-else
-    echo "WARNING: Failed to detect args, use -b to specify"
-fi
-ignition_version=$(disk_ignition_version "${latest_qcow}")
-if [ "${ignition_version}" = "2.2.0" ]; then
-    args="${args} --ignition-version v2"
-fi
-
-if [ "$(id -u)" != "0" ]; then
-    args="${args} -p qemu-unpriv"
-fi
-
-# let's print out the actual exec() call we do
-set -x
+kolaargs = ['kola']
+bn = os.path.basename(qemupath)
+if bn.startswith("rhcos-"):
```

Comment:
Since it'll be a lot easier in python might be worthwhile to gate these additions inside of a check that the flag wasn't already specified, e.x.:
```python
if any([x in args.subargs for x in ["-b", "--distro"]]):
    if bn.startswith("rhcos-"):
        ...
```

Reply:
Your conditional is missing a `not`, right?

Reply:
Ah yes it is.